app/views/registrations/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Admins</h1>
+
+<p><%= link_to "Create new admin", new_registration_path %></p>
+
+<table>
+<% @admins.each do |admin| %>
+  <tr>
+    <td><%= admin.email %></td>
+    <td><%= link_to "Edit", edit_registration_path(admin) %></td>
```

Comment:
Can we only provide an edit link to the logged-in admin's account?  I don't think we want admins editing one another's accounts.